src/main/java/com/cloudant/http/HttpConnection.java

```diff
@@ -68,6 +67,23 @@
 public class HttpConnection {
 
     private static final Logger logger = Logger.getLogger(HttpConnection.class.getCanonicalName());
+
+    private static final String USER_AGENT;
+
+    static {
+        String ua = "java-cloudant-http/unknown";
```

Comment:
Also we have a version here and a user-agent, which is which? Surely a version is eg "1.2.3" and user-agent has all the gory details about platform, OS, etc. Is that the responsibility of the `Version` class or this static block?